src/backend/main.ts

```diff
@@ -166,6 +172,62 @@ export async function bootstrapWorkspace(opts: {
     yDoc.clientID = clientID
   }
 
+  const appPrefixName = appPrefix.toString()
+  const snapshotName = appPrefix.append('32=snapshot').toString()
+  const localYJSUpdate = await persistStore.get('localSnapshot')
+
+  const localTimestamp = await persistStore.get('snapshotTimestamp')
+
+  const timestampInterval = 86400000 //24hr
+  if (!localYJSUpdate || !localTimestamp || Date.now() - NNI.decode(localTimestamp) > timestampInterval) {
+    const interest = new Interest(snapshotName, Interest.CanBePrefix, Interest.MustBeFresh)
+    try {
+      const data = await consume(interest)
+      const targetName = data.name.getPrefix(-1)
+      // /grpPrefix/32=snapshot/54=<vector>/
+      let snapshotData = await fetch(targetName, {
+        verifier: certStorage.verifier, // we have access to the verifier
+        modifyInterest: { mustBeFresh: true },
+        lifetimeAfterRto: 2000,
+        retxLimit: 150, // See Deliveries. 60*1000/(2*200)=150. Default minRto = 150.
```

Comment:
I think this is a problem in my code. I previously used a very long retry limit to get over a limit in NDNts segment fetcher.
Let me check if it is still needed. No need to fix on your side.

FYI: the computation means we want to retry for at least 1 min in case of network issue (60*1000ms), and an Interest is resent per RTT (200ms one way).